arithmetic_arranger_comments.py

```diff
@@ -0,0 +1,68 @@
+def arithmetic_arranger(problems, calculate=False):
+    if len(problems) > 5:
+        return "Error: Too many problems."
+
+    problemss = []
+    for i in range(len(problems)):
+        problemt = problems[i].split()
+        problemss.append(problemt)
+        
+    print(problemss)
+    a = []
+    spacer = '    '
+    line1=''
+    line2=''
+    line3=''
+    line4=''
+    def sp1(x, m):
+        x=' '*(m-len(str(x))+2)+str(x)
+        return x
+
+    def sp2(x, m, y):
+        x=y+' '*(m-len(x)+1)+x
+        return x
+        
+    for i in range(len(problemss)):
+        n1 = problemss[i][0]
+        op = problemss[i][1]
+        n2 = problemss[i][2]
+        l = max(problemss[i], key=len)
+        
+        try:
+            int(n1)
+            int(n2)
+        except:
+            return "Error: Numbers must only contain digits."
+
+        if len(n1) > 4 or len(n2) > 4:
+            return "Error: Numbers cannot be more than four digits."
+            
+        if op == '+':
+            a=int(n1) + int(n2)
+
+        if op == '-':
+            a=int(n1) - int(n2)
+
+        if op == '*' or op == '/':
```

Comment:
This chain of `if` statements should really only result in one possible code path being executed. If you merge the 3 together into a `if/elif/else` chain then it makes the intent a bit more clear here

```
if op == '+': 
  ...
elif op == '-': 
  ...
else: 
  ...
  ```